flax/linen/linear.py

```diff
@@ -688,12 +695,17 @@ def __call__(self, inputs: Array) -> Array:
           -(y_dim - x_dim) % (2 * x_dim)
           for y_dim, x_dim in zip(y.shape[1:-1], scaled_x_dims)
       ]
-      # Divide the padding equaly between left and right. The choice to put
+      # Divide the padding equally between left and right. The choice to put
```

Comment:
Maybe we could move this comment to the `else` branch and have a similar one for the positive branch?

Reply:
I agree, the comments there could be clearer. How about this modified version, with clearer comments for both cases?